conf.d/health_alarm_notify.conf

```diff
@@ -275,6 +278,26 @@ SLACK_WEBHOOK_URL=""
 DEFAULT_RECIPIENT_SLACK=""
 
 
+#------------------------------------------------------------------------------
+# alerta (alerta.io) global notification options
+
+# multiple recipients can be given like this:
+#                  "Environment1 Environment1 ..."
+
+# enable/disable sending alerta notifications
+SEND_ALERTA="YES"
+
+# set your alerta server API url
```

Comment:
Can you add some information for the place this API URL can be obtained from?
Is the API URL the same for all users? You could add it here if it is.